front/app/containers/App/index.tsx

```diff
@@ -268,8 +292,14 @@ const App = ({ children }: Props) => {
     !isIdeaEditPage &&
     !isInitiativeEditPage;
   const { pathname } = removeLocale(location.pathname);
+  const urlSegments = location.pathname.replace(/^\/+/g, '').split('/');
+
   const showFrontOfficeNavbar =
```

Comment:
I think that ideally this logic is in the MainHeader component. That would make it even clearer. Right now, you have to think about checking where we import the componenet, to then realize we don't always show it.